src/extensions/export/bootstrap-table-export.js

```diff
@@ -87,11 +118,11 @@
                             that.$el.tableExport($.extend({}, that.options.exportOptions, {
                                 type: type,
                                 escape: false
-                            }));
+                            }), that.options.data);
                         };
 
                     if (that.options.exportDataType === 'all' && that.options.pagination) {
-                        that.$el.one(that.options.sidePagination === 'server' ? 'post-body.bs.table' : 'page-change.bs.table', function () {
+                        that.$el.one('load-success.bs.table page-change.bs.table', function () {
```

Comment:
Are you sure that this change will work as expected?? This line of code was changed in order to fix an issue related to pagination and the _Export All_ option